tfds_korean/klue_mrc/klue_mrc.py

```diff
@@ -0,0 +1,95 @@
+For more details, see [KLUE Benchmark - NER Task - Overview description](https://klue-benchmark.com/tasks/69/overview/description)
+"""
+
+_CITATION = """
+@misc{park2021klue,
+    title={KLUE: Korean Language Understanding Evaluation},
+    author={Sungjoon Park and Jihyung Moon and Sungdong Kim and Won Ik Cho and Jiyoon Han and Jangwon Park and Chisung Song and Junseong Kim and Yongsook Song and Taehwan Oh and Joohong Lee and Juhyun Oh and Sungwon Lyu and Younghoon Jeong and Inkwon Lee and Sangwoo Seo and Dongjun Lee and Hyunwoo Kim and Myeonghwa Lee and Seongbo Jang and Seungwon Do and Sunkyoung Kim and Kyungtae Lim and Jongwon Lee and Kyumin Park and Jamin Shin and Seonghyun Kim and Lucy Park and Alice Oh and Jungwoo Ha and Kyunghyun Cho},
+    year={2021},
+    eprint={2105.09680},
+    archivePrefix={arXiv},
+    primaryClass={cs.CL}
+}
+"""
+
+_LICENSE = """
+This work is licensed under a [Creative Commons Attribution-ShareAlike 4.0 International License](http://creativecommons.org/licenses/by-sa/4.0/).
+See also [Copyright notice](https://klue-benchmark.com/tasks/69/overview/copyright).
+"""
+
+
+class KlueMrc(tfds.core.GeneratorBasedBuilder):
+    VERSION = tfds.core.Version("1.0.0")
+    RELEASE_NOTES = {"1.0.0": "Initial release."}
+
+    def _info(self) -> tfds.core.DatasetInfo:
+        return tfds.core.DatasetInfo(
+            builder=self,
+            description=_DESCRIPTION,
+            features=tfds.features.FeaturesDict(
+                {
+                    "guid": tfds.features.Text(),
+                    "title": tfds.features.Text(),
+                    "news_category": tfds.features.Text(),
+                    "source": tfds.features.Text(),
+                    "context": tfds.features.Text(),
+                    "question": tfds.features.Text(),
+                    "question_type": tfds.features.Tensor(shape=[], dtype=tf.int64),
+                    "answers": tfds.features.Sequence(
+                        {
+                            "text": tfds.features.Text(),
+                            "answer_start": tfds.features.Tensor(shape=[], dtype=tf.int64),
+                        }
+                    ),
+                    "is_impossible": tfds.features.Tensor(shape=[], dtype=tf.bool),
+                }
+            ),
+            supervised_keys=None,
+            homepage="https://github.com/KLUE-benchmark/KLUE",
+            citation=_CITATION,
+            redistribution_info={"license": _LICENSE},
+        )
+
+    def _split_generators(self, dl_manager: tfds.download.DownloadManager):
+        files = dl_manager.download_and_extract(
+            {
+                "train": "https://raw.githubusercontent.com/KLUE-benchmark/KLUE/4e33d961cec4ecd04073269ef9d26ee33ae5b7cd/klue_benchmark/klue-mrc-v1/klue-mrc-v1_train.json",
+                "dev": "https://raw.githubusercontent.com/KLUE-benchmark/KLUE/4e33d961cec4ecd04073269ef9d26ee33ae5b7cd/klue_benchmark/klue-mrc-v1/klue-mrc-v1_dev.json",
```

Comment:
KLUE는 커밋해시 통일해두고 한꺼번에 다들 교체할까 생각중이예요. 일단은 이렇게 바꿔주세요!

```suggestion
                "train": "https://raw.githubusercontent.com/KLUE-benchmark/KLUE/ab22cd5cfdd6b527a9a4e2d177f9dacb85ddde2c/klue_benchmark/klue-mrc-v1/klue-mrc-v1_train.json",
                "dev": "https://raw.githubusercontent.com/KLUE-benchmark/KLUE/ab22cd5cfdd6b527a9a4e2d177f9dacb85ddde2c/klue_benchmark/klue-mrc-v1/klue-mrc-v1_dev.json",
```